scanpy/preprocessing/docs.py

```diff
@@ -0,0 +1,88 @@
+doc_norm_bulk = """\
+Normalize total counts per cell.
+
+Normalize each cell by total counts over genes, so that every cell has
+the same total count after normalization.
+
+Similar functions are used, for example, by Seurat [Satija15]_, Cell Ranger
+[Zheng17]_ or SPRING [Weinreb17]_.
+
+Parameters
+----------
+data : :class:`~anndata.AnnData`
+    The annotated data matrix of shape `n_obs` × `n_vars`. Rows correspond
+    to cells and columns to genes.
+cell_sum_after : `float` or `None`, optional (default: `None`)
```

Comment:
Can we do `sum_after` instead of `cell_sum_after`? `*counts_per_cell*` shouldn't appear any longer but should be `sum_after`. Can we do "`observation (cell)`" instead of "each cell"?